src/test/kotlin/com/igorwojda/integer/addupto/challenge.kt

```diff
@@ -4,7 +4,9 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
 private fun addUpTo(n: Int): Int {
-    TODO("not implemented")
+    var total = 0
```

Comment:
Please check out: https://kotlinlang.org/docs/reference/collection-aggregate.html

As a side implementation, do you remember the math formula for computing the sum of consecutive integers?